src/libprojectM/projectM.cpp

```diff
@@ -69,11 +69,10 @@ pthread_mutex_t preset_mutex;
 
 projectM::~projectM()
 {
-    void *status;
-
 #ifdef USE_THREADS
+    void *status;
     std::cout << "[projectM] thread ";
-    printf("c");
+    printf("cl");
```

Comment:
maybe just delete these

Reply:
Will try to get to as a part of finding debug printing code in general.